src/components/tabs/list.tsx

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Tabs header container. It is a polymorphic component, which means that, by default, it is an unordered list, but it can be other type of components, and Typescript will infer the correct acceptable attributes.